src/pages/reportSighting/ReportSighting.jsx

```diff
@@ -14,16 +14,28 @@ import Link from '../../components/Link';
 import Button from '../../components/Button';
 import UppyDashboard from '../../components/UppyDashboard';
 import ReportForm from './ReportForm';
+import useGetMe from '../../models/users/useGetMe';
 
 export default function ReportSighting({ authenticated }) {
   const intl = useIntl();
+  const {
+    data: currentUserData,
+    loading: userDataLoading,
+    isFetching: userDataFetching,
```

Comment:
I'm not sure that it is necessary to watch and react to these states. This component is a child of FrontDesk and won't even be rendered if there is no data.

Reply:
Great point. Removed.